### PR TITLE
release-notes 03-03 cut 1

### PR DESCRIPTION
**Pull request by kaarthis**
Please review this.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,27 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2022-03-03
+
+This release is rolling out to all regions - estimated time for completed roll out is 2022-03-16 for public cloud and 2022-03-19 for sovereign clouds.
```
Comment (qpetraroia) [inline CHANGELOG.md:5]: Shouldn't the roll out be 2022-03-14 for public and 22-03-21 for sov? Did we change the release cadence?

**Review by iqmetrix-cloudops — COMMENTED** [review]
Hi, I'm so excited for this release, though I don't see anything regarding GMSA support, please advise

**Review by miwithro — COMMENTED** [review]
Also need to add

"Resolved CVE-2022-0185 in the 2022-02-24 VHD."

File: CHANGELOG.md (hunk 1 above)
Comment (phealy) [reply CHANGELOG.md:5]: When I asked for the last release I was told 13 days and 16 days from the release date - please let me know if we were wrong!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,27 @@
...
+* AKS x OSS Integration Blog Series: This month’s article highlights how to deploy a highly available Redis Cluster to AKS. [Run scalable and resilient Redis with Kubernetes and Azure Kubernetes Service - Microsoft Tech Community](https://techcommunity.microsoft.com/t5/apps-on-azure-blog/run-scalable-and-resilient-redis-with-kubernetes-and-azure/ba-p/3247956). Previous two articles explore storing [Prometheus metrics with Thanos/AKS](https://techcommunity.microsoft.com/t5/apps-on-azure-blog/store-prometheus-metrics-with-thanos-azure-storage-and-azure/ba-p/3067849) and [Cluster monitoring with Prometheus/Grafana/AKS](https://techcommunity.microsoft.com/t5/apps-on-azure-blog/using-azure-kubernetes-service-with-grafana-and-prometheus/ba-p/3020459).
+
+### Release notes
+
+* Behavioral changes
+  * The default VNET address for managed VNETs will change from 10.0.0.0/8 to 10.224.0.0/16 and the default node subnet address will change from 10.224.0.0/12 to 10.224.0.0/16. New clusters will be required to have service and pod CIDR ranges that do not overlap with these new VNET ranges.
+  * Enable-get-volume-stats=true is already true by default on azure file csi driver v1.11.
```
Comment (phealy) [inline CHANGELOG.md:18]: Not quite sure what this item is trying to say - it's already true? It's now set to true?

File: CHANGELOG.md (hunk 1 above)
Comment (qpetraroia) [reply CHANGELOG.md:5]: @phealy Who told you that? I might have been wrong with my dates 😅

Comment (kaarthis) [reply CHANGELOG.md:5]: I will honor the rule of 13 days and 16 days from 03-03 which makes it 03-16 and 03-19 ...

File: CHANGELOG.md (hunk 2 above)
Comment (kaarthis) [reply CHANGELOG.md:18]: Checking with @miwithro

**Review by qpetraroia — APPROVED** [review]

**Pull request merged**